lib/payload/arm/request.rb

```diff
@@ -31,6 +31,7 @@ def filter_by(*args, **data)
 		end
 
 		def all()
+			# TODO: I don't think this applies the @poly variable as intended?
```

Comment:
Yeah I confirmed that if I run the following that it will not filter out `bank_account` payment method types:

```
session = Payload::Session.new(Payload.api_key, Payload.api_url, 1)
cards = session.Card.all()
```

I know we are ignoring polymorphic entities in v1 for now, but just reporting this.